reactjs-films-homework/src/pages/ActorProfile/ActorTitleInfo/ActorTitleInfo.tsx

```diff
@@ -19,6 +21,9 @@ export interface IActorTitleProps {
 const ActorTitle: React.FC<IActorTitle> = ({ props }) => {
   const { birthday, name, profile_path, biography, place_of_birth } = props;
```

Comment:
name and profile_path are strings in the interface, but you check for them in the JSX method. Can they be undefined or empty strings?